src/charm.py

```diff
@@ -366,17 +385,21 @@ def _update_layer_and_restart(self, event) -> None:
             try:
                 if not self.container.exists("/server_data/secret_key"):
                     self.container.exec(["/usr/bin/install.bash"]).wait()
-                environment = {"GRAFANA_DASHBOARD_PATH": "/server_data/grafana_dashboards"}
+                environment = {
+                    "GRAFANA_DASHBOARD_PATH": "/server_data/grafana_dashboards",
+                    "DATABASE_URL": self._database_info_loader(),
```

Comment:
if relation is broken this will be ""? would it be catched and expose an error somewhere in juju?